smdebug/core/utils.py

```diff
@@ -78,6 +82,26 @@ def make_numpy_array(x):
         raise TypeError("_make_numpy_array does not support the" " type {}".format(str(type(x))))
 
 
+def get_aws_region_from_processing_job_arn(processing_job_arn):
```

Comment:
I'm wondering if there's an easier way to get the region. Do we have access to the user's environment with awscli configured? If so, we should get the region from boto3 instead of relying on parsing the processing job ARN.